guides/v2.3/extension-dev-guide/declarative-schema/data-patches.md

```diff
@@ -138,4 +138,4 @@ bin/magento module:uninstall --non-composer Vendor_ModuleName
 ## Will old scripts work in newer versions?
 
 Old scripts will work with new versions of Magento. However, if you want to convert your old scripts to the new format,
-implement `\Magento\Framework\Setup\Patch\PatchVersionInterface`. This interface allows you to specify the setup version of the module in your database. If the version of the module is higher than the version specified in your patch, then the patch is skipped. If the version in the database is equal or lower, then the patch installs.
+implement `\Magento\Framework\Setup\Patch\PatchVersionInterface`. This interface allows you to specify the setup version of the module in your database. If the version of the module is higher or equal than the version specified in your patch, then the patch is skipped. If the version in the database is equal or lower, then the patch installs.
```

Comment:
```suggestion
implement `\Magento\Framework\Setup\Patch\PatchVersionInterface`. This interface allows you to specify the setup version of the module in your database. If the version of the module is higher or equal than the version specified in your patch, then the patch is skipped. If the version in the database is lower, then the patch installs.
```
Hello @slavvka 
What do you think about the above suggestion?

Reply:
Oh yeah, I missed that. Thank you!